backend/src/test/java/gov/cdc/usds/simplereport/service/AddressValidationServiceTest.java

```diff
@@ -0,0 +1,80 @@
+package gov.cdc.usds.simplereport.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.ArgumentMatchers.isA;
+import static org.mockito.Mockito.doNothing;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import com.smartystreets.api.exceptions.SmartyException;
+import com.smartystreets.api.us_street.Candidate;
+import com.smartystreets.api.us_street.Client;
+import com.smartystreets.api.us_street.Lookup;
+import com.smartystreets.api.us_street.Metadata;
+import gov.cdc.usds.simplereport.db.model.auxiliary.StreetAddress;
+import gov.cdc.usds.simplereport.service.errors.InvalidAddressException;
+import java.io.IOException;
+import java.util.ArrayList;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class AddressValidationServiceTest {
+
+  private AddressValidationService s;
+
+  @BeforeEach
+  public void setup() throws SmartyException, IOException {
+    Client client = mock(Client.class);
+    doNothing().when(client).send(isA(Lookup.class));
+    s = new AddressValidationService(client);
+  }
+
+  @Test
+  void throwsExceptionWhenNoAddressesAreFound() {
+    Lookup lookup = mock(Lookup.class);
+    when(lookup.getResult()).thenReturn(new ArrayList<Candidate>());
```

Comment:
```suggestion
    when(lookup.getResult()).thenReturn(List.of());
```

Reply:
This throws a type mismatch error `getResult()` is defined as
```
ArrayList<Candidate> getResult()
```